hack/.import-aliases

```diff
@@ -0,0 +1,3 @@
+{
+	"k8s.io/kubernetes/pkg/kubelet/apis/stats/v1alpha1": "kubeletstatsv1alpha1"
```

Comment:
can you describe how this file is expected to be maintained? a manual mapping like this will drift and be subject to typos, right?